docs/features/logforwarding/outputs/syslog-forwarding.adoc

```diff
@@ -0,0 +1,149 @@
+  If left empty, Syslog will use the whole message as the payload key.
+  Examples:
+
+  {.bar}
+  {.foo.bar.baz}
+  {.foo.bar."baz/with/slashes"}
+
+. `appName`: is APP-NAME part of the syslog-msg header.
+  `appName` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 48. This supports template syntax* to allow dynamic per-event values.
+
+.  `procId`: is `PROCID` part of the syslog-msg header.
+   `procId` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 128. This supports template syntax* to allow dynamic per-event values.
+
+.  `msgId`: is `MSGID` part of the syslog-msg header. This supports template syntax to allow dynamic per-event values. MsgId needs to be specified if using `rfc5424`.  The maximum length of the final values is truncated to 32. This supports template syntax* to allow dynamic per-event values.
+
+. `enrichment`: `None` or `KubernetesMinimal`.
+`None` add no additional enrichment to the record.
+`KubernetesMinimal` adds `namespace_name`, `pod_name`, and `container_name` to the beginning of the message
+body (e.g. `+namespace_name=myproject, container_name=server, pod_name=pod-123, message={"foo":"bar"}+`).
+This may result in the message body being an invalid JSON structure.
+
+
+[NOTE]
+====
+*Template syntax
+
+The `appName`, `msgId` and `procId` can be a combination of static and dynamic values consisting of field paths followed by `||` followed by another field path or a static value.
+A dynamic value is encased in single curly brackets `{}` and MUST end with a static fallback value separated with `||`.
+Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes.
+
+Examples:
+
+  foo-{.bar||"none"}
+  {.foo||.bar||"missing"}
+  foo.{.bar.baz||.qux.quux.corge||.grault||"nil"}-waldo.fred{.plugh||"none"}
+====
+
+=== Defaults value of Syslog fields
+Will be calculating during processing log event if values not set:
+
+==== RFC3164
+
+Format: `+<PRI>TIMESTAMP HOSTNAME TAG: MESSAGE+`
+
+Example: `+<34>Oct 11 22:14:15 mymachine su[1234]: 'su root' failed for lonvick on /dev/pts/+`
+
+|===
+| | journal |application|infrastructure|note
+|AppName|SYSLOG_IDENTIFIER*|namespacePodContainer|.log_source|
+|ProcId|_PID*|N/A|.auditID (if available)|will have no effect in settings
+|MsgId|N/A|N/A|N/A|will have no effect in settings
+|Payloadkey|.message|.message|.message|
+|Severity|N/A|.level|information (6)|
+|Facility|N/A|user (1)|security (13)6|
+|===
+*If `ProcId` available, it will aggregate with `AppName` in the `.tag` field: `appname[procid]`
+
+== RFC5424
+
+Format: `+<PRI> VERSION TIMESTAMP HOSTNAME APP-NAME PROCID MSGID [STRUCTURED-DATA] MESSAGE+`
+
+Example: `+<PRI>1 2024-11-08T14:35:04.123Z ip-10-0-88-196.ec2.internal app-name 1234 ID47 [exampleSDID@32473 iut="3" eventSource="Application" eventID="1011"] This is a sample message+`
```

Comment:
Consider removing structured data from the example.  AFAIK we do not explicitly do anything to populate this or allow a user to control what goes into this field